h2o-algos/src/main/java/hex/rulefit/RuleFitUtils.java

```diff
@@ -131,4 +131,14 @@ static Rule consolidateRule(Rule rule, boolean remove_duplicates) {
 
         return transform.toArray(new Rule[0]);
     }
+
+    static String readRuleId(String ruleId) {
+        // if ruleId is in form after deduplication: "M0T0N1, M0T9N56, M9T34N56", meaning contains ", "
```

Comment:
I would move the comment into the method docstring.

Reply:
8902392